.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -78,6 +78,36 @@ body:
         - Version: [e.g. 91.1]
     validations:
       required: true
+  - type: input
+    id: javascript_runtime_version
+    attributes:
+      label: Javascript Run time version
+      description: |
+        The version of node.js, deno, cloudflare etc... you are currently using.
+    validations:
+      required: true
+  - type: input
+    id: deploy_target
+    attributes:
+      label: Deploy Target
+      description: |
+        Which platform you deploying to? Examples: Vanilla Remix Server, Vercel, CF Worker, Deno, etc..
```

I know I used that term in my comment here, but the "official" term is this 😅 
```suggestion
        Which platform you deploying to? Examples: Remix App Server, Vercel, CF Worker, Deno, etc..
```

---

Also, could we use the examples as placeholders like we do for OS?